components/VoteCard.jsx

```diff
@@ -3,8 +3,21 @@ import styled from "styled-components";
 import Head from "next/head";
 import { Header } from "./Header";
 import ListItem from "./ListItem";
+import Result from "./Result";
 
-export default function VoteCard({ question, inputs }) {
+export default function VoteCard({ question, inputs, setInputs }) {
+  const handleVotePlus = (id) => {
+    const foundInput = inputs.find((input) => input.id === id);
+    foundInput.countYes++;
+    setInputs([...inputs]);
```

Comment:
`inputs` is a react state but it is not allowed to mutate react state.

The find methods gives you a reference to one element of the array. By mutating it (`.countYes++`) you are mutating the state. This can lead to very hard to track down errors.

Refer to the react docs for an detailed explanation: https://beta.reactjs.org/learn/updating-arrays-in-state#updating-objects-inside-arrays

As per the docs a solution using `map` should be preferred (see example in the docs):

```suggestion
    setInputs(inputs.map(input => {
      if (input.id === id) {
        // Create a *new* object with changes
        return { ...input, countYes: input.countYes + 1 };
      } else {
        // No changes
        return input;
      }
    }));
```